src/utils/component.js

```diff
@@ -23,3 +25,16 @@ export function mergeProps(props, defaultProps, inheritedProps, validations) {
     ...filterValidProps(props, validations)
   };
 }
+
+export function areOptionsValid(options) {
+  var isValid = true;
```

Comment:
We should use `let` for consistency.
This aside, we don't need a variable here. You can just `return false` inside `if`, and `return true` by default at the end.